data/json/overmap_terrain.json

```diff
@@ -1602,6 +1602,15 @@
         "known_down" : true
     },{
         "type" : "overmap_terrain",
+        "id" : "standing_stones",
+        "name" : "forest?",
```

Comment:
There's a `cabin_strange` that doesn't bother with a '?' hint. Maybe this one shouldn't either?..